src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -219,6 +220,36 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     action_type = CLIArgumentType(
         help='The action of virtual network rule. Possible value is Allow.'
     )
+
+    enable_vlw_type = CLIArgumentType(
```

Comment:
```suggestion
    enable_alw_type = CLIArgumentType(
```
`vlw` means `version level WORM`, for account, we should change the name to avoid confusion

Reply:
Got it.